client/apps/edit/components/admin/index.coffee

```diff
@@ -60,6 +63,81 @@ module.exports = class EditAdmin extends Backbone.View
     else
       select.setState loading: false
 
+  setupFairProgrammingAutocomplete: ->
+    AutocompleteList = require '../../../../components/autocomplete_list/index.coffee'
+    list = new AutocompleteList @$('#edit-admin-fair-programming')[0],
+      name: 'fair_programming_ids[]'
+      url: "#{sd.ARTSY_URL}/api/v1/match/fairs?term=%QUERY"
+      placeholder: 'Search fair by name...'
+      filter: (res) -> for r in res
+        { id: r._id, value: r.name }
+      selected: (e, item, items) =>
+        @article.save fair_programming_ids: _.pluck items, 'id'
+      removed: (e, item, items) =>
+        @article.save fair_programming_ids: _.without(_.pluck(items, 'id'),item.id)
+    if ids = @article.get('fair_programming_ids')
+      @fairs = []
+      async.each ids, (id, cb) =>
+        request
+          .get("#{sd.ARTSY_URL}/api/v1/fair/#{id}")
+          .set('X-Access-Token': sd.USER.access_token).end (err, res) =>
+            @fairs.push id: res.body._id, value: res.body.name
+            cb()
+      , =>
+        list.setState loading: false, items: @fairs
+    else
+      list.setState loading: false
+
+  setupArtsyAtTheFairAutocomplete: ->
+    AutocompleteList = require '../../../../components/autocomplete_list/index.coffee'
+    list = new AutocompleteList @$('#edit-admin-artsy-at-the-fair')[0],
+      name: 'fair_artsy_ids[]'
+      url: "#{sd.ARTSY_URL}/api/v1/match/fairs?term=%QUERY"
+      placeholder: 'Search fair by name...'
+      filter: (res) -> for r in res
+        { id: r._id, value: r.name }
+      selected: (e, item, items) =>
+        @article.save fair_artsy_ids: _.pluck items, 'id'
+      removed: (e, item, items) =>
+        @article.save ffair_artsy_ids: _.without(_.pluck(items, 'id'),item.id)
+    if ids = @article.get('fair_artsy_ids')
+      @fairs = []
+      async.each ids, (id, cb) =>
+        request
+          .get("#{sd.ARTSY_URL}/api/v1/fair/#{id}")
+          .set('X-Access-Token': sd.USER.access_token).end (err, res) =>
+            @fairs.push id: res.body._id, value: res.body.name
+            cb()
+      , =>
+        list.setState loading: false, items: @fairs
+    else
+      list.setState loading: false
+
+  setupAboutTheFairAutocomplete: ->
```

Comment:
Looks good! We might want to think about breaking this admin view into multiple parts now (maybe a view per-admin section e.g. AdminFeaturingView, AdminMetadataView). Also probably an opportunity to DRY up this autocomplete init code, but that one is actually less offensive to me b/c it's mostly config.